### PR TITLE
fix: make grid columns span correctly

### LINKED ISSUE #638: fix(grid): columnEnd is off by one
#### Expected Behaviour

Given a grid 2x2; I expect to define a view to end in the last column to have `"columnEnd": 2`.

#### Actual Behaviour
"columnEnd" needs to be set to "3" for the view to take the whole width.

### PR DESCRIPTION
**Pull request by henrikbossart**
## What does this pull request change?
the grid positioning is referencing the grid lines, not the columns. Therefor I increment the end by one to reference the "ending line" of the column
## Issues related to this change
closes #638

### CODE REVIEW

**Comment by henrikbossart** [comment]
That didn't quite turn out how I wanted...

**Comment by henrikbossart** [comment]
Slightly tweaked an example, still not 100%, but I think the grid works as intended now, so the examples shouldn't be blocking the fix for other use-cases.

File: packages/dm-core-plugins/src/grid/GridPlugin.tsx [hunk 1]
```diff
@@ -21,6 +21,9 @@ export const GridPlugin = (props: TGridPluginConfig): React.ReactElement => {
       rowGap={config.size.rowGap}
       columnGap={config.size.columnGap}
     >
+      {config.items.forEach(() => (
+        <div style={{ backgroundColor: 'hotpink' }}></div>
+      ))}
```
Comment (soofstad) [inline packages/dm-core-plugins/src/grid/GridPlugin.tsx:26]: Whats this? :thinking:

**Review by soofstad — APPROVED** [review]

File: packages/dm-core-plugins/src/grid/GridPlugin.tsx (hunk 1 above)
Comment (henrikbossart) [inline packages/dm-core-plugins/src/grid/GridPlugin.tsx:26]:
```suggestion
```

**Pull request merged**